doc/release-notes/8578-support-extended-iso-639-languages.md

```diff
@@ -0,0 +1,11 @@
+The Controlled Vocabulary Values list for the metadata field Language in the Citation block can now be extended to include roughly 7920 ISO 639-3 values.
+
+To be added to the 6.4 release instructions:
+
+Update the Citation block, to incorporate the additional controlled vocabulary for languages:
```

Comment:
This should be in the guides somewhere as well, not just in a release note.

Reply:
added more